test/concerns/sorted_relations_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class SortedRelationsTest < ActiveSupport::TestCase
+  test 'sorted_relations_for shoud persist for N to N relation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.